newdocs/package.json

```diff
@@ -16,7 +18,7 @@
     "react-dom": "^16.8.4",
     "remark": "^12.0.0",
     "remark-collapse": "^0.1.2",
-    "remark-sources": "^1.0.0"
+    "remark-sources": "RasaHQ/remark-sources#7236b62"
```

Comment:
opened this PR: https://github.com/unlight/remark-sources/pull/1 to get the changes in the parent repo